### ISSUE TITLE
How to use this in ng nativescript

### ISSUE THREAD

**Issue by Habeeb-mohamed** [EddyVerbruggen/nativescript-calendar#8]
I try to import that by it fails in typescript compilation. With below error.

`error TS2307: Cannot find module 'nativescript-calendar'.`

Import code 

`import Calendar = require("nativescript-calendar");`

**Comment by SjurWarEagle** [comment]
Did you add
"nativescript-calendar": "1.0.7",
to your package.json and ran npm install?

I'm running
`import {createEvent} from "nativescript-calendar";
import {CreateEventOptions} from "nativescript-calendar";

let options:CreateEventOptions = {
	title: "SomeText",
	startDate: new Date(1000),
	endDate: new Date(1000),
	calendar: {name: "ACalendar"}
	//uses first calender it finds calendar: {name: undefined}
};

createEvent(options)
`
wihtout problems.

**Comment by Habeeb-mohamed** [comment]
I am using 
` "nativescript-calendar": "^1.1.1",`
While discussing on slack. I got the information that Visual studio code has certain issues in module.

Using any import i have same error, even i try like below implementation
`import { createEvent } from "nativescript-calendar";`

I did the following things to fix the error

1. Add typings key in package.json in node_modules/nativescript-calendar.  `"typings":"calendar.d.ts",`
2. Remove ` export module` from both calendar-common.d.ts and calendar.d.ts
3. Copy calendar-common.js code to calendar.ios.js and .android.js

After doing this issue was solved.
